attestation-agent/attester/src/tdx/mod.rs

```diff
@@ -187,10 +186,17 @@ impl Attester for TdxAttester {
         let extend_data: [u8; 48] = pad(&event_digest);
 
         log::debug!(
-            "TDX Attester: extend RTRM{rtmr_index}: {}",
+            "TDX Attester: extend RTMR{rtmr_index}: {}",
             hex::encode(extend_data)
         );
 
+        #[cfg(not(feature = "tdx-attest-dcap-ioctls"))]
+        std::fs::write(
+            Path::new(TDX_RTMR_PATH).join(format!("rtmr{rtmr_index}:sha384")),
+            extend_data,
+        )
+        .context("TDX Attester: failed to extend RTMR")?;
+
```

Comment:
I found that we have multiple `cfg` conditions here for ioctl/non-ioctl. Could you help to make two separate same-name functions and just call it here but enable one of them with feature flag?

Reply:
I'll re-think this along with the updated `runtime_measurement_extend_available()` logic.